media-sound/jack/jack-4.1_pre20230723.ebuild

```diff
@@ -1,40 +1,58 @@
-# Copyright 2021-2023 Gentoo Authors
+# Copyright 2021-2024 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=8
 
-PYTHON_COMPAT=( python3_{9..11} )
+PYTHON_COMPAT=( python3_{10..12} )
 PYTHON_REQ_USE="ncurses"
 DISTUTILS_SINGLE_IMPL=1
+DISTUTILS_USE_PEP517=setuptools
 
 inherit distutils-r1
 
 DESCRIPTION="A frontend for several cd-rippers and mp3 encoders"
 HOMEPAGE="https://github.com/jack-cli-cd-ripper/jack https://www.home.unix-ag.org/arne/jack/"
-GIT_COMMIT_HASH="5f3e5f43f476b0d506144c103eb8d4edf76fc5de" # branch "python3-mb"
-SRC_URI="https://github.com/jack-cli-cd-ripper/jack/archive/${GIT_COMMIT_HASH}.tar.gz -> ${P}.tar.gz"
+GIT_COMMIT_HASH="795fba0ef1859b39b3bb41ca162ab402177808a5" # branch "python3-mb"
+GIT_DOC_HASH="4303994b67222639ee4c3f55b214020b2f5c75f4"
+SRC_URI="
+	https://github.com/jack-cli-cd-ripper/jack/archive/${GIT_COMMIT_HASH}.tar.gz -> ${P}.tar.gz
+	https://github.com/jack-cli-cd-ripper/jack/raw/${GIT_DOC_HASH}/example.etc.jackrc
+	https://github.com/jack-cli-cd-ripper/jack/raw/${GIT_DOC_HASH}/jack.man"
```

Comment:
Even so seems weird to get documentation from an old point in history. Maybe better to look at what debian does instead or not include them?

https://github.com/jack-cli-cd-ripper/jack/commit/af77231a716993ec54e462d406cfd3bff026af71

Reply:
Those files seem to provide value. But mgorny is right, they should carry the version in the filename to avoid conflicts. I'd simply rename them using the arrow operator of SRC_URI.

Reply:
FWIW, debian does use the following man page https://sources.debian.org/src/jack/4~git20230906.795fba0-2/debian/jack.1/ which seems very similar, if not identical, to the one obtained by this ebuild.

Reply:
@madmartin 

please change the SRC_URI so that the files are versioned. For example "https://github.com/jack-cli-cd-ripper/jack/raw/${GIT_DOC_HASH}/jack.man -> jack-4.0.0_pre20201210.man". Then it's good to go.

Reply:
@Flowdalic files have version suffix now.
@mgorny split into 2 commits.

Better now?